src/CONST.js

```diff
@@ -1165,6 +1165,7 @@ const CONST = {
         CARD_EXPIRATION_DATE: /^(0[1-9]|1[0-2])([^0-9])?([0-9]{4}|([0-9]{2}))$/,
         PAYPAL_ME_USERNAME: /^[a-zA-Z0-9]{1,20}$/,
         ROOM_NAME: /^#[a-z0-9à-ÿ-]{1,80}$/,
+        COUNTRY_NAME_WITH_ONLY_ALPHABETS: /[-'().&\s]/g,
```

Comment:
Let's do this the other way around and exclude all non-alphabetic characters. We have to also remember there's a Spanish locale
```suggestion
        NON_ALPHABETIC_AND_NON_LATIN_CHARS: /[^a-zA-ZÀ-ÿ]/g,
```